perses/rjmc/geometry.py

```diff
@@ -193,11 +199,17 @@ def logp_reverse(self, top_proposal, new_coordinates, old_coordinates):
 
                 #propose a bond and calculate its probability
                 bond = self._get_relevant_bond(atom, bond_atom)
-                logp_r = self._bond_logp(internal_coordinates[0], bond, top_proposal.beta)
+                bond_k = bond.type.k*units.kilocalorie_per_mole/units.angstrom**2
```

Comment:
AMBER bond force constants may be off by a factor of two. Check what @swails does here carefully?
https://github.com/ParmEd/ParmEd/blob/master/parmed/structure.py#L2002-L2003